Admin/BaseMediaAdmin.php

```diff
@@ -50,6 +53,15 @@ public function __construct($code, $class, $baseControllerName, Pool $pool, Cate
     /**
      * {@inheritdoc}
      */
+    public function configureRoutes(RouteCollection $collection)
+    {
+        $collection->add('multi_upload', 'upload/multi');
```

Comment:
just asking, but is it correct, that we don't have `{provider}`, but for the ajax route we have?